draft-ietf-quic-transport.md

```diff
@@ -1739,6 +1739,14 @@ network, an endpoint verifies the following when an ACK_ECN frame is received:
 * The increase in ECT(0) and ECT(1) counters MUST be no greater than the number
   of packets newly acknowledged that were sent with the corresponding codepoint.
 
+* If the increase in the ECT(0), ECT(1) and CE counters are greater than
+  the number of newly acknowledged packets, and the received ACK's
+  smallest acknowledged is larger than the previous largest acknowledged,
```

Comment:
As I pointed out, this isn't ideal and will lead to ECN being disabled in some cases.  I don't know how much we are going to want to worry about that though.